### PR TITLE
build: melos

### PR DESCRIPTION
**Pull request by altynbek132**
Add mobx_lint_flutter_test package to melos.yaml
update melos dev dependency to version 5.2.1

### CODE REVIEW

File: melos.yaml [hunk 1]
```diff
@@ -16,7 +17,3 @@ scripts:
     exec: dart run $MELOS_ROOT_PATH/tool/expose_version.dart
   generate:
     run: cd mobx_examples && dart run build_runner watch --delete-conflicting-outputs
-
```
Comment (pavanpodila) [inline melos.yaml:19]: This is required to work locally. We should not remove this

**Review by pavanpodila — CHANGES_REQUESTED** [review]
Let's keep the command. Looks good otherwise

File: melos.yaml (hunk 1 above)
Comment (altynbek132) [reply melos.yaml:19]: this options has been removed in melos 3.1.1, isn't it?
https://github.com/invertase/melos/blob/melos-v3.1.1/docs/guides/migrations.mdx#L52

Comment (pavanpodila) [reply melos.yaml:19]: Awesome .. didn't know that ...

**Review by pavanpodila — APPROVED** [review]

**Comment by pavanpodila** [comment]
@all-contributors add @altynbek132 for code

**Pull request merged**